### PR TITLE
add a page to show available themes

### PR DESCRIPTION
**Pull request by SherOnDale**
image
image

### CODE REVIEW

File: website/pages/themes.vue [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+        justify-items: center;
+      }
+    }
+
+    &__card {
+      width: 20rem;
+    }
+  }
+}
+</style>
```
Comment (egoist) [inline website/pages/themes.vue]: let's use PostCSS only 😂

Comment (SherOnDale) [reply website/pages/themes.vue]: oh lol i didnt know there is no sass-loader xD

**Comment by krmax44** [comment]
The side bar should probably be removed :smile:

https://github.com/egoist/saber/blob/c82dd4344f4f9b6820f91668475525f44abbe132/website/src/layouts/themes.vue#L4

Also on phones, with Themes added to the navbar, a more responsive solution is probably needed. The padding of the cards is also a bit off, especially on phones.

**Comment by diomed** [comment]
Tailsaw theme should be added

**Pull request closed without merging**